.gitignore

```diff
@@ -9,3 +9,4 @@ daffodil-debug.txt
 daffodil-debugger*
 *.debug
 datafile-hex
+*infoset*
```

Comment:
Infoset is too common of a term in the context of daffodil to ignore it here.

This needs removed and addressed in #67